userComponent/src/main/kotlin/com/gravatar/app/usercomponent/domain/usecase/SelectUserAvatarUseCase.kt

```diff
@@ -1,19 +1,17 @@
 package com.gravatar.app.usercomponent.domain.usecase
 
-import com.gravatar.app.clock.AppClock
 import com.gravatar.app.usercomponent.data.AvatarCacheBusterStorage
 import com.gravatar.app.usercomponent.domain.repository.UserRepository
 
 internal class SelectAvatarUseCase(
     private val userRepository: UserRepository,
     private val avatarCacheBusterStorage: AvatarCacheBusterStorage,
-    private val clock: AppClock,
 ) : SelectUserAvatar {
 
     override suspend fun invoke(avatarId: String): Result<Unit> {
         return userRepository.selectAvatar(avatarId)
             .onSuccess {
-                avatarCacheBusterStorage.saveAvatarCacheBuster(clock.now().toString())
+                avatarCacheBusterStorage.saveAvatarCacheBuster(avatarId)
```

Comment:
Great idea! I tried to find a case that could create some issues, but I haven't been able to come up with anything. Awesome! 💯